scripts/eksa_version.sh

```diff
@@ -65,7 +65,16 @@ function eksa-version::get_next_eksa_version_for_ancestor() {
         # bump one minor version and use patch 0
         
         latest_tag=$(git describe --tags "$(git rev-list --tags='v*.*.*' --max-count=1)")
-        
+
+        # Query to check if the latest_tag is of n'th branch.
+        # For ex, if the latest release is of n-1 branch, we need to ignore that tag and find the next branch.
+        latest_minor=$(echo "${latest_tag}" | cut -d "." -f 2)
```

Comment:
```suggestion
        latest_minor=$(( $(echo "${latest_tag}" | cut -d "." -f 2) + 1 ))
```
nit: could combine it into one line and accurately name the var